modules/post/android/gather/wireless_ap.rb

```diff
@@ -78,9 +82,9 @@ def parse_network_block(block)
   end
 
   def parse_option(block, token, strip_quotes = true)
-    if strip_quotes and result = block.match(/^\s#{token}="(.+)"$/)
+    if strip_quotes && ((result = block.match(/^\s#{token}="(.+)"$/)))
```

Comment:
Am I being dumb here or is there no reason for the double parenthesis? single would be fine right?

Reply:
Yeah, shouldn't be needed. I'll not manually changing Rubocop's output though to reduce review time. i.e. this PR is all automated, and hasn't been hand-tuned afterwards.